.jshint

```diff
@@ -1,7 +1,28 @@
 {
-  "undef": true,
-  "unused": true,
-  "camelcase": true,
+  "undef": false,
+  "unused": false,
   "jquery": true,
-  "browser": true
-}
+  "browser": true,
+  "asi": true,
+  "boss": true,
+  "curly": false,
+  "eqeqeq": false,
+  "eqnull": true,
+  "esnext": true,
+  "expr": true,
+  "forin": false,
+  "immed": false,
+  "laxbreak": true,
+  "newcap": false,
+  "noarg": true,
+  "node": true,
+  "nonew": true,
+  "plusplus": false,
+  "strict": false,
+  "evil": true,
+  "-W018": false,
+  "-W058": false,
+  "-W008": false,
+  "-W054": false,
+  "-W004": false
+}
```

Comment:
Add an empty line at the end of the file. (You can set your editor to do this for you)